cluster/terraform/k8s/json/argo_namespace_iam_policy.json

```diff
@@ -16,6 +16,14 @@
             "Resource": [
                 "*"
             ]
+        },
+        {
+          "Effect": "Allow",
+          "Action": ["s3:PutObject", "s3:GetObject"],
+          "Resource": [
+            "arn:aws:s3:::curi-prod-workflows",
+            "arn:aws:s3:::curi-prod-workflows/*"
+          ]
```

Comment:
I already ran the terraform to create the buckets, this is only for prod so there isn't a test overlay